### DISCUSSION TITLE
Additional Outputs for Loaders

### DISCUSSION THREAD

**Discussion by FarrunDragon** [FakeFishGames/Barotrauma#8709] [Ideas and suggestions]
I would like at least an output for percentage of ammo left in the loader (so you know when it's low). This would make it easier than having to manually check each one.

**Comment by Prolbo** [comment]
"Is empty" would be enough. But % is also fine.

**Reply by mygamingaccount** [reply]
From % to is_empty is only a not component. % is clearly better
